src/FileManager.js

```diff
@@ -0,0 +1,19 @@
+class FileManager {
+  constructor() {
+    this.cohorts = []
+  }
+
+  addCohort(cohort) {
+    this.cohorts.push(cohort)
+  }
+
+  removeCohortByName(name) {
+    this.cohorts = this.cohorts.filter((cohort) => cohort.CohortName !== name)
+  }
+
+  findCohortByName(name) {
```

Comment:
I like that change! Having this search method live in FileManager() instead of Cohort() makes a lot of sense. 

I wonder if you'd want to use .find() instead of .filter() - the former would return the element you are looking for without it being wrapped in [ ]. 
That said, changing this would only really make sense if the plan is to make it impossible for multiple cohorts to have the same name.